### PR TITLE
created header file

### PR DESCRIPTION
**Pull request by cookandy**
created header.php and referenced it from all front-end pages

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (Arcanemagus) [inline header.php]: You shouldn't be re-declaring the doctype in the middle of the document, the same goes for the <html> tags.

**Comment by Arcanemagus** [comment]
There's no reason to wrap the <includes> in blank lines on either side.

File: header.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		<div class="container">
+			<div class="navbar navbar-fixed-top">
+				<div class="navbar-inner">
+					<a href="index.php"><div class="logo hidden-phone"></div></a>
+				<ul class="nav">
+					<li><a href="index.php"><i class="fa fa-home fa-2x" data-toggle="tooltip" data-placement="bottom" title="Home" id="home"></i></a></li>
+					<li><a href="history.php"><i class="fa fa-history fa-2x" data-toggle="tooltip" data-placement="bottom" title="History" id="history"></i></a></li>
+					<li><a href="users.php"><i class="fa fa-users fa-2x" data-toggle="tooltip" data-placement="bottom" title="Users" id="users"></i></a></li>
+					<li><a href="stats.php"><i class="fa fa-area-chart fa-2x" data-toggle="tooltip" data-placement="bottom" title="Stats" id="stats"></i></a></li>
+					<li class="active"><a href="charts.php"><i class="fa fa-bar-chart fa-2x" data-toggle="tooltip" data-placement="bottom" title="Charts" id="charts"></i></a></li>
```
Comment (Arcanemagus) [inline header.php]: This line right here should tell you the reason this wasn't done before. With the way you currently have this implemented the charts page will _always_ be listed as the active tab. You need some sort of dynamic system in there to only mark the real active tab. You can probably do this based on the currently requested page.

Comment (cookandy) [inline header.php]: Yes, you are right - I didn't even notice the different class on the active page.  I'll look into dynamically picking that up via js.  Thx.

Comment (Arcanemagus) [inline header.php]: You should be able to do it all server side in the PHP, no need to do it via JS (although that would work as well, you might as well have all the code in the same place.)

**Comment by cookandy** [comment]
created new pull with changes

**Pull request closed without merging**